infrastructure/lambda/public-api.yaml

```diff
@@ -111,7 +111,7 @@ paths:
       x-amazon-apigateway-integration:
         httpMethod: "POST"
         uri:
-          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${IssueCredentialFunction.Arn}/invocations
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${IssueCredentialFunction.Arn}:live/invocations
```

Comment:
The alias will be the same, we are keeping the same convention so both can work together

Reply:
Changing api spec and permissions if using canaries would cause errors and a rollback (this is what happened in orange - kbv) but no issue if deploying this without canaries. Canary changes rely on the live version of the lambdas as well so it will be useful to have this change in already